### ISSUE TITLE
[Bug]: internal error: pipeline context has done during runing tpcc test

### ISSUE THREAD

**Issue by aressu1985** [matrixorigin/matrixone#9789]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

<img width="1096" alt="image" src="https://github.com/matrixorigin/matrixone/assets/47846308/d72f764f-729f-4613-baac-b987c3e154ef">

log is located in s3. path is:
s3://mo-checkin-regression/result/9774/

the merge trigger job is:
https://github.com/matrixorigin/matrixone/actions/runs/5133850849/jobs/9237369432

**Comment by aressu1985** [comment]
https://github.com/matrixorigin/matrixone/actions/runs/5133850849/jobs/9237369432

**Comment by dongdongyang33** [comment]
in process

**Comment by dongdongyang33** [comment]
in process

**Comment by aressu1985** [comment]
fixed:nightly-d4d74e7
